src/follower_npc.c

```diff
@@ -1655,10 +1657,13 @@ static const u8 *const FollowerNPCHideMovementsSpeedTable[][4] =
 
 void ScriptHideNPCFollower(struct ScriptContext *ctx)
 {
+    if (!PlayerHasFollowerNPC())
```

Comment:
```suggestion
    if (!FNPC_ENABLE_NPC_FOLLOWERS || !PlayerHasFollowerNPC())
```
I think (but I'm not sure) this would speed it up even further by reducing the amount of function calls